server.js

```diff
@@ -41,7 +41,7 @@
     mime.define({
         'application/json' : ['czml', 'json', 'geojson', 'topojson'],
         'model/vnd.gltf+json' : ['gltf'],
-        'model/vnd.gltf.binary' : ['bgltf'],
+        'model/vnd.gltf.binary' : ['bgltf', 'glb'],
```

Comment:
This will likely change, https://github.com/KhronosGroup/glTF/issues/412.

I added it to the task list in this PR.